### PR TITLE
Challenge1 - add function printHelloName

### PR DESCRIPTION
**Pull request by milicadatwork**
add function printHelloName

### CODE REVIEW

**Review by vesnaatwork — APPROVED** [review]
this is great! you could have also solve this by calling the function inside the for loop ie.
`for (let i=0; i<5; i++) {
     printHelloName("Milica");
}`

File: hello.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// Function to compute the product of p1 and p2
+function printHelloName(name) {
+    for (let i = 0; i < 5; i++) {
+        const element = 5[i];
```
Comment (stefanworkco) [inline hello.js]: what does `const element = 5[i];` do? Also, `return name;` is not needed, it will work without that. But looks great, overall

**Review by tamaracvorovic-work — APPROVED** [review]
You can change your commit per Vesna's and Stefan's comments. But looks great, overall. :)

**Comment by milicadatwork** [comment]
Added the fix regarding your comments. Thank you

**Pull request merged**